scripts/install_ostis.sh

```diff
@@ -1,69 +1,74 @@
 #!/bin/bash
 
+APP_ROOT_PATH=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && cd .. && pwd)
+PLATFORM_PATH="${APP_ROOT_PATH}/ostis-web-platform"
+WORKING_PATH=$(pwd)
+PYTHON_PATH=${APP_ROOT_PATH}/problem-solver/py/services
```

Comment:
```suggestion
PYTHON_PATH="${APP_ROOT_PATH}"/problem-solver/py/services
```
Should be wrapped in quotes everywhere.